opacus/tests/batch_memory_manager_test.py

```diff
@@ -36,28 +38,28 @@ def _init_training(self, **data_loader_kwargs):
 
         return model, optimizer, data_loader
 
-    # @given(
-    # num_workers=st.integers(0, 4),
-    # pin_memory=st.booleans(),
-    # )
+    @given(
+        num_workers=st.integers(0, 4),
+        pin_memory=st.booleans(),
+    )
+    @settings(deadline=10000)
     def test_basic(
         self,
-        num_workers: int = 0,
-        pin_memory: bool = True,
+        num_workers,
+        pin_memory,
```

Comment:
why remove type hints?

Reply:
went a little too far when removing default values, thanks for noticing